src/server/index.jsx

```diff
@@ -104,14 +104,20 @@ server
 
       const headHelmet = Helmet.renderStatic();
 
-      const styleTags = sheet.getStyleElement();
+      let inlineCss = '';
+      try {
+        // eslint-disable-next-line prefer-destructuring
+        inlineCss = renderToString(sheet.getStyleElement()).match(
+          /^<style (?:[^>]+)>([\S\s.]+)<\/style>$/,
+        )[1];
```

Comment:
This throws away the styled components ```<style data-styled-components="jLeGVa hmqJYW jHyUsA kzqBtt jNOQvG aJDbs dNlZTz iIqwzV iQnGvB gSfTCk cYVMCV dVfLHk bzGli kVHsaM PDRnW gMmsmd bkhVuy emBiDe dvYMEz diCwNi lbsaqU fUFvnt bwxfrq kSWLHu eqvZjV jvhmFc" data-reactroot="">```
I have no idea what theyre used for, but its probably something